packages/freezed/lib/src/freezed_generator.dart

```diff
@@ -370,6 +371,10 @@ Read here: https://github.com/rrousselGit/freezed/tree/master/packages/freezed#t
         configs['union_key']?.toString() ??
         'runtimeType';
 
+    final fallbackUnion =
+        annotation.getField('fallbackUnion')?.toStringValue() ??
+            configs['fallback_union']?.toString();
```

Comment:
I don't think that specifying a fallback key from the configs makes sense.

Reply:
I thought about the case when someone wants to just use `fallback()` factory everywhere. Or at least in places that factory is created. The same way that unionKey can be globally defined in config. But that would require not throwing error when there is no constructor with specified key (as you pointed out below). So I can remove it.